lib/bootstrap.js

```diff
@@ -45,13 +47,39 @@ async function runCLI(cli, commandIsUsed) {
     let args;
     const helpFlagExists = isFlagPresent(process.argv, 'help');
     const versionFlagExists = isFlagPresent(process.argv, 'version');
+    const nodeArgsExists = isFlagPresent(process.argv, 'node-args');
 
     if (helpFlagExists) {
         cli.runHelp(process.argv);
         return;
     } else if (versionFlagExists) {
         cli.runVersion();
         return;
+    } else if (nodeArgsExists) {
+        args = cmdArgs(core, { stopAtFirstUnknown: false, partial: true });
+        const [, , ...rawArgs] = process.argv;
+        const cliArgs = [];
+        const nodeArgs = [];
+        let isNodsArg = false;
+
+        for (const value of rawArgs) {
+            if (value === '--node-args') {
+                isNodsArg = true;
+            } else if (isNodsArg) {
+                isNodsArg = false;
+                nodeArgs.push(...value.split(' '));
+            } else {
+                cliArgs.push(value);
+            }
+        }
+
+        try {
+            const childProcess = execa('node', [...nodeArgs, cliPath, ...cliArgs], { stdio: 'inherit' });
```

Comment:
At the moment `execa`is a dev dependency. You should move it inside the `dependencies`

Reply:
It’s already moved ;)